lib/capybara/webkit/driver.rb

```diff
@@ -415,11 +416,26 @@ def deprecate_and_replace_with_config(deprecated_method, config_syntax = depreca
     def encode_args(args)
       args.map do |arg|
         if arg.is_a?(Capybara::Webkit::Node)
-          { ELEMENT: arg.native }.to_json
+          { 'ELEMENT' => arg.native }.to_json
         else
           arg.to_json
         end
       end
     end
+
+    def decode_result(result)
+      case result
```

Comment:
Now that we're returning values that aren't part of the native JSON domain, I think I'd rather:

* Consistently return a `Hash`
* Have a `type` field specifying how to process the return value (`array`, `object`, or `element`)
* Have a `value` field with the actual value

It feels like a cleaner abstraction and eliminates the possible bug where somebody tries to return a regular object with a field called `ELEMENT`.

What do you think?